pkg/operator/bootstrap.go

```diff
@@ -102,8 +102,8 @@ func (c *Controller) Bootstrap() error {
 		Spec: imageregistryv1.ImageRegistrySpec{
 			ManagementState: mgmtState,
 			LogLevel:        2,
-			Storage:         imageregistryv1.ImageRegistryConfigStorage{},
-			Replicas:        1,
+			Storage:         platformStorage,
+			Replicas:        replicas,
 			HTTPSecret:      fmt.Sprintf("%x", string(secretBytes[:])),
 			RolloutStrategy: string(rolloutStrategy),
```

Comment:
A note for others - we separately set the RolloutStrategy to `Recreate` if the registry is using block storage on OpenStack.